subsys/usb/device_next/class/usbd_midi.c

```diff
@@ -0,0 +1,786 @@
+/*
+ * Copyright (c) 2024 Titouan Christophe
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT zephyr_usb_midi
+
+#include <zephyr/drivers/usb/udc.h>
+#include <zephyr/sys/byteorder.h>
+#include <zephyr/sys/ring_buffer.h>
+#include <zephyr/usb/class/usbd_midi.h>
+#include <zephyr/usb/usbd.h>
+
+#include "usbd_uac2_macros.h"
```

Comment:
Do you still need to include it?

Reply:
Yes, there are a handful of defines that are reused from there (to name a few):
* `AUDIO` class code https://github.com/zephyrproject-rtos/zephyr/blob/main/subsys/usb/device_next/class/usbd_uac2_macros.h#L51
* `MIDISTREAMING` subclass code https://github.com/zephyrproject-rtos/zephyr/blob/main/subsys/usb/device_next/class/usbd_uac2_macros.h#L57
* `AUDIOCONTROL` subclass code https://github.com/zephyrproject-rtos/zephyr/blob/main/subsys/usb/device_next/class/usbd_uac2_macros.h#L55